README.md

```diff
@@ -6,9 +12,18 @@ docker-credential-helpers is a suite of programs to use native stores to keep Do
 
 Go to the [Releases](https://github.com/docker/docker-credential-helpers/releases) page and download the binary that works better for you. Put that binary in your `$PATH`, so Docker can find it.
 
-### Building from scratch
+## Building
+
+You can easily build the credential helpers using Docker:
+
+```shell
+# create builder
+$ docker buildx create --use
```

Comment:
Does this _require_ a container builder? If it's just single arch, it should work without, I think?

Reply:
Yes required because we are building against multi-platforms (osxkeychain, wincred, pass)

Reply:
ah, right, it's building all of them

what's the easiest way to build only one?